addons/binding/org.openhab.binding.mysensors/test/iVersionMessage.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+serialPort=$1
+
+echo "0;0;3;0;2;0" > $serialPort
```

Comment:
Unexpected test for me... should this be checked in, if so what would the proper directory be.